connect-button/src/test/java/com/ifttt/connect/ui/ConnectButtonTest.java

```diff
@@ -15,6 +17,11 @@
 
     private final Activity activity = Robolectric.buildActivity(TestActivity.class).get();
 
+    @Before
+    public void setUp() {
+        ActivityScenario.launch(TestActivity.class);
```

Comment:
Why do we need to do this?

Reply:
The work manager required initialization in the `TestActivity` onCreate. So this is to launch the activity so that `onCreate` is called.

Reply:
The `ActivityController` returned from `buildActivity` can be used to control lifecycle events, if you call `create()` before `get()`, it should move the activity to onCreate.